File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py
class Solution(object):
    def uniqueOccurrences(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        res = []
        for i in list(set(arr)):
            res.append(arr.count(i))

        return len(res) == len(set(res))

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.go
func uniqueOccurrences(arr []int) bool {
    m := map[int]int{}
    for _, v := range arr {
        m[v] += 1
    }

    occ := map[int]bool{}
    for _, v := range m {
        if occ[v] {
            return false
        }

        occ[v] = true
    }

    return true
}
